app/code/Magento/Bundle/Pricing/Price/ConfiguredPrice.php

```diff
@@ -48,6 +48,10 @@ class ConfiguredPrice extends CatalogPrice\FinalPrice implements ConfiguredPrice
      * @var ConfiguredPriceSelection
      */
     private $configuredPriceSelection;
+    /**
+     * @var \Magento\Bundle\Pricing\Price\DiscountCalculator
```

Comment:
Please preserve an empty row and do not use FQPN. Please import the class name.
```suggestion

    /**
     * @var DiscountCalculator
```